<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>register_window</class>
 <widget class="QMainWindow" name="register_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="register_rno">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ADMIN REGISTER</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>590</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>REGISTER</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>280</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>PIN:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_pin"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>85</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>PASS:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_pass">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>410</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>TOTAL CAP.(IN TON):-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_cap"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>345</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>REGION:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_rno_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>NAME:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>215</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>DISTRICT:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_district"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>ADDRESS:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_address"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>470</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>CONTACT NO:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_contact"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="reg_admin_msg">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>620</y>
        <width>341</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_9">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>530</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>USERNAME:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="register_username"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>VENDOR REGISTER</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_19">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>320</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_27">
       <item>
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>PIN:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_pin"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_20">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_28">
       <item>
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>ADDRESS:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_address"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_21">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>510</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_29">
       <item>
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>CONTACT NO:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_contact"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_22">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>255</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_30">
       <item>
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>DISTRICT:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_distric"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>580</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>REGISTER</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_23">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>385</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_31">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>AFFILATED COLD STORAGE REGION:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_24">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_32">
       <item>
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>NAME:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_name"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_25">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>125</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_33">
       <item>
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>PASS:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_pass"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_27">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>450</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_35">
       <item>
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>USERNAME:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_user"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>610</y>
        <width>641</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>FARMER REGISTER</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>570</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>SUBMIT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_49">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>620</y>
        <width>691</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_34">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>480</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_50">
       <item>
        <widget class="QLabel" name="label_47">
         <property name="text">
          <string>CONTACT NO:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_contact_3"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_31">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>420</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_47">
       <item>
        <widget class="QLabel" name="label_44">
         <property name="text">
          <string>USERNAME:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_user_3"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_29">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_45">
       <item>
        <widget class="QLabel" name="label_42">
         <property name="text">
          <string>ADDRESS:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_address_3"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_33">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>95</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_49">
       <item>
        <widget class="QLabel" name="label_46">
         <property name="text">
          <string>PASS:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_pass_3"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_26">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>355</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_43">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>AFFILATED COLD STORAGE REGION:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_32">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_48">
       <item>
        <widget class="QLabel" name="label_45">
         <property name="text">
          <string>NAME:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_name_3"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_30">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_46">
       <item>
        <widget class="QLabel" name="label_43">
         <property name="text">
          <string>PIN:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_pin_3"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_28">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>225</y>
        <width>620</width>
        <height>59</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_44">
       <item>
        <widget class="QLabel" name="label_41">
         <property name="text">
          <string>DISTRICT:-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vendor_register_distric_3"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>610</y>
        <width>611</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuREGISTER">
    <property name="title">
     <string>REGISTER</string>
    </property>
   </widget>
   <addaction name="menuREGISTER"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
